### PR TITLE
refactor(theme): use whiskers

### PR DESCRIPTION
**Pull request by vdbe**

### CODE REVIEW

File: themes/latte/latte.tmuxtheme
Comment (vdbe) [inline themes/latte/latte.tmuxtheme:1]: With the "new" colors:
image

File: templates/catppuccin-flavor.tmuxtheme.tera [hunk 1]
```diff
@@ -0,0 +1,26 @@
+---
+whiskers:
+  version: "2.1.0"
+  matrix:
+    - flavor
+  filename: "themes/{{flavor.identifier}}/{{flavor.identifier}}.tmuxtheme"
```
Comment (sgoudham) [inline templates/catppuccin-flavor.tmuxtheme.tera:6]: Wouldn't it make more sense to have a directory structure like:

```
themes/
├─ catppuccin_latte.tmuxtheme
├─ catppuccin_frappe.tmuxtheme
├─ catppuccin_macchiato.tmuxtheme
├─ catppuccin_mocha.tmuxtheme
```

Feels like the flatter version is nicer, is there a practical reason for nesting it further?

File: themes/latte/latte.tmuxtheme
Comment (sgoudham) [reply themes/latte/latte.tmuxtheme:1]: Looks good!

**Review by sgoudham — CHANGES_REQUESTED** [review]

File: templates/catppuccin-flavor.tmuxtheme.tera
Comment (sgoudham) [inline templates/catppuccin-flavor.tmuxtheme.tera:1]: Our informal convention thus far for the file name is `<port name>.tera`, which in this case would be `tmux.tera`. I think it'd be nice to update it but I'm happy to approve this PR even without the rename :+1:

File: templates/catppuccin-flavor.tmuxtheme.tera (hunk 1 above)
Comment (vdbe) [reply templates/catppuccin-flavor.tmuxtheme.tera:6]: Was from when the matrix included accents

**Review by sgoudham — APPROVED** [review]

**Pull request merged**